### PR TITLE
Multiple UI Improvements

### PR DESCRIPTION
**Pull request by asaadmahmoodspin**
Hey guys have a look, have fixed the styleguide errors for my code and also fixed the refs error in my previous PR. Please re-review. These changes include work for the following tickets:

* MM-1819 - Adding strong for user name in Private messages.
* MM-1820 - Adding strong tags for the channel intro text.
* MM-1654 - Adding max height to channel description popover and allowing scroll.
* MM-1939 - Adjusting style of unread posts above bar.
* MM-1831 - Making channel names bold for modals
* MM-1839 - Showing preview footer when user is hovering over the left/right arrows for previews

### CODE REVIEW

File: web/react/components/view_image.jsx [hunk 1]
```diff
@@ -109,6 +109,26 @@ module.exports = React.createClass({
             }
         );
 
+        if (this.refs.previewArrowLeft.getDOMNode) {
```
Comment (crspeller) [inline web/react/components/view_image.jsx]: Should this be `if (this.refs.previewArrowLeft) {` ?

**Comment by asaadmahmoodspin** [comment]
Yeah I think that's right Chris, so I just updated that in the PR.

File: web/react/components/post_list.jsx [hunk 2]
```diff
@@ -416,9 +417,9 @@ module.exports = React.createClass({
                         <div className='channel-intro'>
                             <h4 className='channel-intro__title'>Beginning of {uiName}</h4>
                             <p className='channel-intro__content'>
-                                Welcome to {uiName}!
+                                Welcome to <strong>{uiName}</strong>!
                                 <br/><br/>
-                                {'This is the first channel ' + strings.Team + 'mates see when they'}
+                                This is the first channel {strings.Team} mates see when they
```
Comment (rgarmsen2295) [inline web/react/components/post_list.jsx]: Is the added space between {strings.Team} and mates intentional?  Believe it should be teammates with no space.

File: web/react/components/post_list.jsx [hunk 3]
```diff
@@ -453,7 +454,7 @@ module.exports = React.createClass({
 
                     var createMessage;
                     if (creatorName !== '') {
-                        createMessage = 'This is the start of the ' + uiName + ' ' + uiType + ', created by ' + creatorName + ' on ' + utils.displayDate(channel.create_at) + '.';
+                        createMessage = (<span>This is the start of the <strong>{uiName}</strong> {uiType} created by <strong>{creatorName}</strong> on <strong>{utils.displayDate(channel.create_at)}</strong></span>);
```
Comment (rgarmsen2295) [inline web/react/components/post_list.jsx]: Is the removal of the comma before "created by" intentional?

File: web/react/components/post_list.jsx (hunk 2 above)
Comment (asaadmahmoodspin) [inline web/react/components/post_list.jsx]: Agreed. will update that.

File: web/react/components/post_list.jsx (hunk 3 above)
Comment (asaadmahmoodspin) [inline web/react/components/post_list.jsx]: I don't think so, might've missed that in the changes, updating this too.

**Comment by rgarmsen2295** [comment]
+1

**Comment by asaadmahmoodspin** [comment]
Sovling the merge conflicts and also adding the following improvement:
* MM-1752 - Making PM profile pics rounded

**Comment by crspeller** [comment]
Looks good to me.

**Comment by jwilander** [comment]
+1

**Pull request merged**